frontend/public/components/dashboards-page/overview-dashboard/capacity-card.tsx

```diff
@@ -0,0 +1,104 @@
+import * as React from 'react';
+import * as _ from 'lodash-es';
+
+import * as plugins from '../../../plugins';
+import {
+  DashboardCard,
+  DashboardCardBody,
+  DashboardCardHeader,
+  DashboardCardTitle,
+} from '../../dashboard/dashboard-card';
+import { CapacityBody, CapacityItem } from '../../dashboard/capacity-card';
+import { withDashboardResources, DashboardItemProps } from '../with-dashboard-resources';
+import { humanizePercentage, humanizeDecimalBytesPerSec, humanizeBinaryBytesWithoutB } from '../../utils';
+import { getInstantVectorStats, getRangeVectorStats, GetStats } from '../../graphs/utils';
+import { CapacityQuery } from './capacity-query-types';
+
+const defaultQueries = {
+  [CapacityQuery.CPU_USED]: '((sum(node:node_cpu_utilisation:avg1m) / count(node:node_cpu_utilisation:avg1m)) * 100)[60m:5m]',
+  [CapacityQuery.MEMORY_TOTAL]: 'sum(kube_node_status_capacity_memory_bytes)',
+  [CapacityQuery.MEMORY_USED]: '(sum(kube_node_status_capacity_memory_bytes) - sum(kube_node_status_allocatable_memory_bytes))[60m:5m]',
+  [CapacityQuery.STORAGE_TOTAL]: 'sum(node_filesystem_size_bytes)',
+  [CapacityQuery.STORAGE_USED]: '(sum(node_filesystem_size_bytes) - sum(node_filesystem_free_bytes))[60m:5m]',
+  [CapacityQuery.NETWORK_TOTAL]: 'sum(avg by(instance)(node_network_speed_bytes))',
+  [CapacityQuery.NETWORK_USED]: 'sum(node:node_net_utilisation:sum_irate)',
+};
+
+const getLastStats = (response, getStats: GetStats): React.ReactText => {
```

Comment:
can we move this function in the utils too?? This will be required by all the dashboard cards trying to show a single stat value.

Reply:
@cloudbehl @rawagner I'd suggest a follow-up PR to move this function into `console-shared` package.

Our current priority is to get all the dashboard related PRs (and associated extension types) merged first.

Reply:
I agree, will create a follow-up PR